nemo/collections/multimodal/speech_llm/data/lhotse_s2s_dataset.py

```diff
@@ -290,6 +465,7 @@ def get_step_by_time(text_start_time):
             "instructions": None,
             "tokens": target_texts_merge,  # used in _reconfigure_and_process_inference_batch
             "target_texts_merge": target_texts_merge,  # used in prepare_llm_input
+            "source_texts_merge": source_texts_merge,  # used in prepare_llm_input
```

Comment:
do we include any delay in source_texts_merge channel compared to the source_speech? i think we need it since streaming ASR prediction requires some delay to work, right?

Reply:
For now, I think let's do an initial run without delay. We can produce a baseline first and then move on with ablations.

Reply:
ok can u add a comment of "TODO" on where to add delay if we want to

Reply:
Added.